packages/openapi-to-graphql/src/types/operation.ts

```diff
@@ -41,16 +50,23 @@ export type DataDefinition = {
    */
   subDefinitions: DataDefinition | { [fieldName: string]: DataDefinition }
 
-  links: { [key: string]: LinkObject }
+  // GraphQL-related:
+
+  // The potential name of the GraphQL type if it is created
+  graphQLTypeName: string
```

Comment:
Calling this `otName` was a misnomer as the type we created could have been multiple different types, specifically object, array, or enum. 

I think this could also be a bit confusing as this is just a potential name, not the exact GraphQL type name, but I think it is an improvement over what we had before. Same for the other changes below.

Reply:
Agreed.